src/adapters/WstEthOracle.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import {IStEth} from "../interfaces/IStEth.sol";
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+
+/// @title WstEthOracle
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice wstETH/ETH exchange rate price feed.
+/// @dev This contract should only be used as price feed for `ChainlinkOracle`.
+contract WstEthOracle is AggregatorV3Interface {
```

Comment:
```suggestion
contract WstEthChainlinkAdapter is AggregatorV3Interface {
```

The rationale is we need to indicate it's about being compatible to Chainlink's API